lib/performance_platform.rb

```diff
@@ -5,7 +5,9 @@ class Base
     attr_reader :headers
 
     def initialize(token = nil)
-      token || abort('No token set. Note that this task should only be executed in production.')
+      return unless Rails.env.production?
```

Comment:
Nice. Much better to be explicit, just in case we do set the token in another environment